style/swift/README.md

```diff
@@ -5,6 +5,7 @@ Swift
 
 * Keep up with the Objective-C style guide above. Will highlight differences
   here.
+* Limit lines to 100 characters
```

Comment:
@gylaz I tried to reference that in the PR message/commit message. As I noted and as Tony said, 80 characters is unrealistic most of the time because of the verbosity of the APIs we're working with.